crates/ubrn_bindgen/src/bindings/react_native/gen_typescript/templates/macros.ts

```diff
@@ -110,7 +110,7 @@
 
 {%- macro call_async(obj_factory, callable) -%}
         await uniffiRustCallAsync({
```

Comment:
So I think this is declared to take a `() => bigint` but it seems like it is passing an object to it as the first arg?  Did I miss something?

Reply:
I misread the template, nevermind